source/example/F0.Analyzers.Example/CodeAnalysis/CodeRefactorings/ObjectInitializerExample.cs

```diff
@@ -64,6 +65,11 @@ public NestedStruct GetInitialized()
 			};
 		}
 
+		public IEnumerable<string> GetCollection()
+		{
+			return new List<string>();
```

Comment:
You can use `Enumerable.Empty<string>()` here.

Reply:
Here, actually the `new List<string>();` is intentional:
I've used this _Example_ also for manual testing, and we need an `ObjectCreationExpression` with the `new` keyword to confirm, that the "Create Object Initializer" is no longer registered with Collections.